eclipse-collections-code-generator/src/main/resources/api/list/mutablePrimitiveList.stg

```diff
@@ -151,6 +151,11 @@ allMethods(type) ::= <<
  */
 Mutable<name>List sortThis();
 
+/**
+ * Randomly permutes this list mutating its contents and returns the same list (this).
+ */
+Mutable<name>List shuffleThis();
```

Comment:
This should have a default implementation unless you want this PR to wait for a major release.

Reply:
Will pull the implementation up to MutablePrimitveList interface as default